packages/audio-assets/scripts/generate-audio-manifest.mjs

```
@@ -0,0 +1,86 @@
+    },
+  };
+}
+
+function main() {
+  const manifest = buildManifest();
+  fs.writeFileSync(OUTPUT_FILE, JSON.stringify(manifest, null, 2) + "\n", "utf-8");
+  console.log(`Generated ${OUTPUT_FILE}`);
+  console.log(`Clips: ${manifest.clips.length}`);
+}
+
+if (import.meta.url === `file://${process.argv[1].split(path.sep).join("/")}` || process.argv[1] === url.fileURLToPath(import.meta.url)) {
```

ESLint

[error] 'process' is not defined.

(no-undef)